<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="accountInfoGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>800</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Blackadder ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Account Information</string>
    </property>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>41</y>
       <width>47</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Account Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="countryEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>45</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Account Country</string>
     </property>
    </widget>
    <widget class="QLabel" name="countryLabel">
     <property name="geometry">
      <rect>
       <x>246</x>
       <y>43</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailEdit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>45</y>
       <width>220</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Email Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="emailLabel">
     <property name="geometry">
      <rect>
       <x>493</x>
       <y>43</y>
       <width>47</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="settingGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>800</width>
      <height>190</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Blackadder ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QCheckBox" name="autoSignupCheckBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>42</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto-bid after signup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoLoginCheckBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>92</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto-bid after login</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="sendMsgToTelegramCheckBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>142</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Send messages to telegram</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signupTimesEdit">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="signupIntervalTimeEdit">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>96</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="signupIntervalLabel">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>minutes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="autoBidIntervalTimeEdit">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>146</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="autoBidIntervalTimeLabel">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>140</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>minutes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>43</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of signup</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>92</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Signup interval time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Bid interval time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>43</y>
       <width>47</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>times</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Blackadder ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Blackadder ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Blackadder ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auto-bid</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>451</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Blackadder ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
